azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/ApplicationInsightsProperties.java

```diff
@@ -205,22 +224,17 @@ public void setInProcess(InProcess inProcess) {
        */
       private boolean developerMode = false;
       /** Endpoint address. */
-      private String endpointAddress = TransmissionNetworkOutput.DEFAULT_SERVER_URI;
```

Comment:
We want to allow `null` so we can tell if a custom endpoint is set in application.properties